src/mlpack/tests/load_save_test.cpp

```diff
@@ -184,6 +184,87 @@ BOOST_AUTO_TEST_CASE(LoadTransposedCSVTest)
   // Remove the file.
   remove("test_file.csv");
 }
+/**
+  *Make sure ColVec can be loaded
+**/
+BOOST_AUTO_TEST_CASE(LoadColVecCSVTest)
+{
+  fstream f;
+  f.open("test_file.csv", fstream::out);
+
+  for( int i = 0; i < 8; ++i)
+    f << i << endl;
+
+  f.close();
+
+  arma::colvec test;
+  BOOST_REQUIRE(data::Load("test_file.csv", test, false) == true);
+
+  BOOST_REQUIRE_EQUAL(test.n_cols, 1);
+  BOOST_REQUIRE_EQUAL(test.n_rows, 8);
+
+  for(size_t i = 0; i < 8; ++i)
+    BOOST_REQUIRE_CLOSE(test[i], (double) (i), 1e-5);
+
+  //Remove the file
+  remove("test_file.csv");
+}
+
+/**
+  *Make Sure Load throws Exception when trying
+  *to load a Matrix in ColVec 
+  *and RowVec
+**/
+#ifdef DEBUG
```

Comment:
It would be nice to add a comment here pointing out why this test only works under debug mode.